content/general-docs/introduction.adoc

```diff
@@ -3,11 +3,81 @@
 
 ifndef::include-only-once[]
 :root-path: ../
+:imagesdir: {root-path}/_images
+:includedir: {root-path}/_images
 include::{root-path}_config.adoc[]
 endif::[]
 
 == Overview
-Overview of {THIS_STANDARD}.
+
+=== Motivation
+
+Accurate sensor simulation that can replicate physical phenomena is necessary for safety evaluation of AD/ADAS in virtual space. 
+
+However, previous simulations were heavily limited by computational power and available memory, and while they appeared visually believable, they were not physically accurate.
+As a result, previous simulations were primarily used during the early stages of development, where precise physical modeling was not necessarily required. 
+
+With increased computing power and streamlined simulation techniques, virtual testing plays a more powerful role at all stages of system development.
+This correlates with the growing need to accurately represent the real world in simulations, so-called digital twins, especially as the need for physical sensor simulation increases. 
+
+One fundamental step to describing the real world virtually is a full physical description of all objects, including their material properties and their 3D geometries.
+
+It is extremely important to standardize the representation of data and attributes, such as scenarios, road data, and asset data, necessary for AD/ADAS simulation in order to promote the circulation of these data and to obtain consistent results among different systems. 
+
+Standardization activities have been conducted for scenarios and road data using OpenSCENARIO, OpenDRIVE, OpenCRG, and other standards.
```

Comment:
we need to use the name "ASAM" at the beginning: 
ASAM OpenSCENARIO; ASAM OpenDRIVE,..

Reply:
Thanks, I fixed it.